packages/rich-text-editor/docs/RichTextEditor.stories.tsx

```diff
@@ -26,4 +26,21 @@ export const Default = () => {
     </>
   )
 }
+
+Default.storyName = "Default (Kaizen Demo)"
+
+export const WithControls = () => {
```

Comment:
FWIW I don't think we need to keep the example with no toolbar, but lets address this later by replacing these with a sticker sheet when we add the moods.